### ISSUE TITLE
Music time progress bar stay still on 0:00

### ISSUE THREAD

**Issue by LudoBike** [quodlibet/quodlibet#1953]
Hi, I report a bug on the music time progress bar, when I start music it stay still on 0:00 however if I change the time manually the bar work well 

Here some info on my OS
```bash
# ludovic @ Tardis in ~ [16:48:17] 
$ uname -a
Linux Tardis 4.5.4-1-ARCH #1 SMP PREEMPT Wed May 11 22:21:28 CEST 2016 x86_64 GNU/Linux

# ludovic @ Tardis in ~ [16:48:19] 
$ quodlibet --version
Quod Libet 3.6.1
<quod-libet-development@googlegroups.com>
Copyright © 2004-2016 Joe Wreschnig, Michael Urman, Iñigo Serna, Steven Robertson, Christoph Reiter, Nick Boultbee...

```

**Comment by urielz** [comment]
I've come across this issue as well, perhaps half a dozen times in the last year or so. To me it happened at random times while I was doing something else. I couldn't pin down the steps to reproduce it. (OS X QL 3.x)

**Comment by lazka** [comment]
Does playback stop as well?

**Comment by LudoBike** [comment]
No the music is in playing but the time stay still

**Comment by lazka** [comment]
@urielz does playback also continue in your case?

**Comment by urielz** [comment]
yes it does..

**Comment by lazka** [comment]
ok, thanks

**Comment by kaffeeundsalz** [comment]
Just wanted to say that I've occasionally observed this, too. I can't add much information though, since @urielz has already pointed it all out. Playback continues, can't reproduce steps. Currently QL 3.6.1 on OS X 10.11.5 EL Capitan, but the bug showed up in earlier versions, too.

**Comment by pierregermain** [comment]
I have this also when I change the tags of a song from the command line.

**Comment by CreamyCookie** [comment]
At the moment I can reproduce this. It worked even after I restarted Quod Libet. All I did was to start playing a song, then after a few seconds hold the seekbar with the left mouse for yet a few more seconds. Now, the music time is always 0:00 even when switching to a different song. I can make it work again by dragging the seekbar to something other than 0:00. 

One thing I noticed (which may or may not be related to this bug) is that the music time sometimes lags behind a lot. I just noticed that the music was playing but the time was still 0:00, then it suddenly jumped to 0:06 and started working as usual.
